scripts/services/migrations.ts

```diff
@@ -210,6 +210,33 @@ export const migration: T.ExpectedExports.migration = compat.migrations
           throw new Error("Cannot downgrade");
         },
       },
+      "0.18.0": {
+        up: compat.migrations.updateConfig(
+          (config: any) => {
+            config.advanced["protocol-zero-conf"] = false;
+            config.advanced["protocol-option-scid-alias"] = false;
```

Comment:
Worth noting that zero-conf requires option-scid-alias. I know migrations isn't the right place to handle that, but probably needs to be added in the config validation scripts.

The check would be something like `!protocol-zero-conf || protocol-option-scid-alias` which would need to resolve to true for the config to validate properly.

Reply:
We have a validation for `protocol-opiton-scit-alias` and `protocol-no-anchors` in [setConfig.ts](https://github.com/Start9Labs/lnd-startos/blob/update/0.18.0/scripts/services/setConfig.ts#L34-L47).

Toast errors aren't really optimal, but I believe this is the best that can be done for dynamic config requirements until 0.3.6.

getConfig.ts also notes the requirement in the description [here](https://github.com/Start9Labs/lnd-startos/blob/update/0.18.0/scripts/services/setConfig.ts#L34-L47)

Reply:
Cool yeah this covers my issue.